client/changelist/change.go

```diff
@@ -37,15 +37,15 @@ type TUFChange struct {
 // TUFRootData represents a modification of the keys associated
 // with a role that appears in the root.json
 type TUFRootData struct {
-	Keys     data.KeyList `json:"keys"`
-	RoleName string       `json:"role"`
+	Keys     data.KeyList  `json:"keys"`
+	RoleName data.RoleName `json:"role"`
 }
 
 // NewTUFChange initializes a TUFChange object
-func NewTUFChange(action string, role, changeType, changePath string, content []byte) *TUFChange {
+func NewTUFChange(action string, role data.RoleName, changeType, changePath string, content []byte) *TUFChange {
 	return &TUFChange{
 		Actn:       action,
-		Role:       role,
+		Role:       role.String(),
```

Comment:
`TUFChange` should probably use a `RoleName` here, and thread the type through the rest of the change operations.